tls/tls_srv.c

```diff
@@ -1094,9 +1095,8 @@ ttls_parse_client_hello(TlsCtx *tls, unsigned char *buf, size_t len,
 	if (ttls_ciphersuite_uses_ecdh(tls->xfrm.ciphersuite_info) ||
 	    ttls_ciphersuite_uses_ecdhe(tls->xfrm.ciphersuite_info))
 	{
-		unsigned char pf = tls->hs->ecdh_ctx.point_format;
 		ttls_ecdh_init(&tls->hs->ecdh_ctx);
-		tls->hs->ecdh_ctx.point_format = pf;
+		tls->hs->ecdh_ctx.point_format = tls->hs->ecdh_ctx.point_format;
```

Comment:
This optimization introduces a bug: `.point_format` is cleared by `ttls_ecdh_init()`. We use `.point_format` to store point format during ClientHello parsing.

Reply:
Oops, my bad. Thank you!